claim/README.md

```diff
@@ -54,14 +54,58 @@ With `sudo`:
 sudo netdata-claim.sh -token=TOKEN -rooms=ROOM1,ROOM2 -url=https://app.netdata.cloud
 ```
 
-Hit **Enter**. The script should return `Agent was successfully claimed.`.
+Hit **Enter**. The script should return `Agent was successfully claimed.`. If the claiming script returns errors, see
+the [troubleshooting information](#troubleshooting).
 
 > Your node may need up to 60 seconds to connect to Netdata Cloud after finishing the claiming process. Please be
 > patient!
 
-If the claiming script returns errors, see the [troubleshooting information](#troubleshooting).
+### Claim an Agent running in Docker
 
-### Claiming through a proxy
+The claiming process works with Agents running inside of Docker containers. You can use `docker exec` to run the
+claiming script on containers already running, or append the claiming script to `docker run` to create a new container
+and immediately claim it.
+
+#### Running Agent containers
+
+Claim a _running Agent container_ by appending the script offered by Cloud to a `docker exec ...` command:
+
+```bash
+docker exec -it netdata-claim.sh -token=TOKEN -rooms=ROOM1,ROOM2 -url=https://app.netdata.cloud
```

Comment:
This is missing either a "Container ID" or "Container Name" (_if Docker was run with `docker run --name ...`_) as the first argument.  See:

```#!sh
$ docker exec --help

Usage:	docker exec [OPTIONS] CONTAINER COMMAND [ARG...]
```

Reply:
Good catch. I'd missed that. Added a `netdata` name to the command and referenced it above.